### ISSUE TITLE
CJS dist files don't display in the Chrome "source" panel

### ISSUE THREAD

**Issue by akagomez** [stealjs/steal#1382]
Removing this line at the end of the file seems to help: 

```
//# sourceMappingURL=viewmodel.js.map
```

**Comment by matthewp** [comment]
Sorry I missed this. Do you remember what this was about? If so can you provide some more context? Does this mean that you had a CommonJS file with a source map file and you had to remove the comment to get the source to show up?

**Comment by akagomez** [comment]
@matthewp Yeah, if I remember correctly, that was the case. Can't provide much context beyond that. Twas a long time ago.
